### PR TITLE
new-feature:add k3net

### PR DESCRIPTION
**Pull request by IloveKanade**
The net module has been modified according to the specification

### CODE REVIEW

**Review by pengsven — APPROVED** [review]

File: packages.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+pandoc
```
Comment (drmingdrmer) [inline packages.txt:1]: Where is `pandoc` used? I do not remember... 🤔

**Review by drmingdrmer — APPROVED** [review]
LGTM

File: packages.txt (hunk 1 above)
Comment (IloveKanade) [reply packages.txt:1]: This is generated by the template file "tmpl/_building/tmpl/requirements.txt"
 I don't know what the purpose of this file is😗

Comment (drmingdrmer) [reply packages.txt:1]: Yeah it's a template that has some useless demo things there.
Forget what you are taught, trust your judgement.

Comment (IloveKanade) [reply packages.txt:1]: > Yeah it's a template that has some useless demo things there.
> Forget what you are taught, trust your judgement.

i got it😁

**Pull request merged**